src/components/ui/Button.tsx

```diff
@@ -69,6 +69,7 @@ export interface ButtonProps
     VariantProps<typeof buttonVariants> {
   iconOnly?: boolean
   icon?: LucideIcon
+  iconClassname?: string
```

Comment:
yeah I can agree with adding this. Feels like there are some cases where we want to override default icon styling.